### DISCUSSION TITLE
How to change make the default template of a Workflow to resource type

### DISCUSSION THREAD

**Discussion by akldm007** [argoproj/argo-workflows#9449] [Q&A]
I had a WorkflowTemplate which has a template of a **resource** type to deploy a customize CR.
And I am using a Workflow to trigger the WorkflowTemplate by **workflowTemplateRef**.
But the workflow will also always trying to start a container "main" which lead to the failure, do you know how to make a workflow ref a resource type from WorkflowTemplate?

**Comment by terrytangyuan** [comment]
Check your controller and main container log to see what exactly failed.
